CondFormats/GeometryObjects/interface/PTrackerParameters.h

```diff
@@ -115,6 +115,8 @@ class PTrackerParameters
   TIBItem tib;
   TIDItem tid;
   TOBItem tob;
+
+  std::vector<int> pDetIdShifts;
```

Comment:
Hi Salvatore - For me it is up to AlCa, who is the originator of this no-schema-evolution policy.


On Jan 26, 2015, at 2:36 PM, Salvatore Di Guida <notifications@github.com>
 wrote:

> In CondFormats/GeometryObjects/interface/PTrackerParameters.h:
> 
> > @@ -115,6 +115,8 @@ class PTrackerParameters
> >    TIBItem tib;
> >    TIDItem tid;
> >    TOBItem tob;
> > +
> > +  std::vector<int> pDetIdShifts;
> 
> @ianna @davidlange6 I checked that it is not yet in production, so it is still ok to do the change.
> I let @ggovi comment further.
> 
> —
> Reply to this email directly or view it on GitHub.
>